outport/process/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -140,6 +137,25 @@ func (aap *alteredAccountsProvider) processMarkedAccountData(
 	return nil
 }
 
+func (aap *alteredAccountsProvider) getAlteredAccountFromUserAccounts(userEncodedAddress string, userAccount state.UserAccountHandler) *outportcore.AlteredAccount {
+	alteredAccount := &outportcore.AlteredAccount{
+		Address:  userEncodedAddress,
+		Balance:  userAccount.GetBalance().String(),
+		Nonce:    userAccount.GetNonce(),
+		UserName: string(userAccount.GetUserName()),
+	}
+
+	if core.IsSmartContractAddress(userAccount.AddressBytes()) {
```

Comment:
does this works also on a metachain SC?

Reply:
yes, is working also from the metachain smart contracts.